rosario_angelica/country.js

```diff
@@ -42,23 +44,25 @@ async function searchCountry() {
   }
 }
 
-// Function to display the details of the selected country
+
 function displayCountryDetails(details) {
-  const detailsHTML = details.map(detail => `<p><strong>${detail.label}:</strong> ${detail.value}</p>`).join('');
+  const detailsHTML = details.map(detail => `<p><strong>${detail.label}:
+  </strong> ${detail.value}</p>`).join('');
   countryDetails.innerHTML = `<h3>Country Details</h3>${detailsHTML}`;
 }
 
-// Function to fetch and display countries in the same region
 async function fetchRegionCountries(region) {
   try {
     const response = await fetch(`${BASE_URL}/region/${region}`);
     const data = await response.json();
 
     const countryNames = data.map(country => country.name.common);
-    const countriesHTML = countryNames.map(name => `<li>${name}</li>`).join('');
-    regionCountries.innerHTML = `<h3>Countries in the ${region} Region</h3><ul>${countriesHTML}</ul>`;
+    const countriesHTML = countryNames.map(name => `<li>
+    ${name}</li>`).join('');
```

Comment:
Same goes here! Please add 1 tab indentation before the code ${name} in line 61:
1. not adheres readability: ❌
![image](https://github.com/PUP-BSIT/exercise-14-sixth_sense/assets/146615920/54dd34bc-cd23-4841-bebd-3e3f70f1c902)
2. edit your code just like in line 50: ✅
![image](https://github.com/PUP-BSIT/exercise-14-sixth_sense/assets/146615920/35f3f31f-65e6-459b-97f8-b64d2f74064a)

Reply:
After you fix this, you're good to go!